deps/llvm-ver.make

```diff
@@ -1,3 +1,7 @@
+ifeq ($(USE_SYSTEM_LLVM), 1)
+LLVM_VER := $(shell $(LLVM_CONFIG_HOST) --version)
```

Comment:
The main issue in the past has been getting the right staging order. The deps makefiles aren't supposed to see USE_SYSTEM for that dependency, as that sometimes corrupts the makefile rules. Even though they shouldn't be getting called, that can cause issues and confuse make when it tries to parse the file. We might need to introduce a post-deps Make.inc that gets only used by Julia, vs. a smaller one that get used by deps.